src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -153,9 +167,18 @@ protected function printPendingMessagesMessage(ReceiverInterface $receiver, Symf
         }
     }
 
-    protected function getReceiver(): ReceiverInterface
+    protected function getReceiver(/* string $name */): ReceiverInterface
     {
-        return $this->receiver;
+        if (\func_num_args() < 3 && __CLASS__ !== \get_class($this) && __CLASS__ !== (new \ReflectionMethod($this, __FUNCTION__))->getDeclaringClass()->getName() && !$this instanceof \PHPUnit\Framework\MockObject\MockObject && !$this instanceof \Prophecy\Prophecy\ProphecySubjectInterface) {
+            @trigger_error(sprintf('The "%s()" method will have a new "string $name" argument in version 5.3, not defining it is deprecated since Symfony 5.2.', __METHOD__), E_USER_DEPRECATED);
+        }
+        $name = 2 < \func_num_args() ? \func_get_arg(2) : '';
+
+        if (!$this->failureTransports->has($name)) {
+            throw new InvalidArgumentException('The failure transport with name "%s" was not found', $name);
```

Comment:
Missing a sprintf here - so the `$name` is being passed as arg 2 to the exception